packages/suite/src/actions/wallet/selectedAccountActions.ts

```diff
@@ -215,11 +225,13 @@ const actions = [
     ROUTER.LOCATION_CHANGE,
     SUITE.SELECT_DEVICE,
     SUITE.UPDATE_SELECTED_DEVICE,
+    SUITE.TOR_STATUS,
     metadataActions.setAccountLoaded.type,
     metadataActions.setAccountAdd.type,
     accountsActions.createAccount.type,
     accountsActions.removeAccount.type,
     accountsActions.updateAccount.type,
+    accountsActions.updateSelectedAccount.type,
```

Comment:
Yes, it does not make sense. I was trying that since the first time after reloading I was finding the `sectedAccount` with initial state, so I was trying to load it again, but yes it is very hacky... 

Removed by https://github.com/trezor/trezor-suite/pull/6796/commits/bb849b3475bd35a6ea04899ca135b9f71d4f63e1